inference-engine/src/mkldnn_plugin/nodes/mkldnn_def_conv_node.cpp

```diff
@@ -675,9 +672,15 @@ void MKLDNNDeformableConvolutionNode::initSupportedPrimitiveDescriptors() {
 
     impl_desc_type impl_type;
     const int simd_w = mayiuse(cpu::x64::avx512_common) ? 16 : 8;
-    if (group != 1 || (((getInputShapeAtPort(0).getStaticDims()[1] / group) % simd_w != 0)
-    || ((getOutputShapeAtPort(0).getStaticDims()[1] / group) % simd_w != 0))) {
+
+    if (getInputShapeAtPort(WEI_ID).getDims()[1] == Shape::UNDEFINED_DIM ||
+            getInputShapeAtPort(WEI_ID).getDims()[0] == Shape::UNDEFINED_DIM ||
+            defConvAttr.group != 1 ||
+            (getInputShapeAtPort(WEI_ID).getDims()[1] % simd_w != 0)  // in_channels_per_gr !% simd_w
+            || ((getInputShapeAtPort(WEI_ID).getDims()[0] / defConvAttr.group) % simd_w != 0)) {  // out_channels_per_gr !% simd_w
```

Comment:
`defConvAttr.group != 1` condition is a WA, because of inadopted jit implementation. In fixed version the second condition with channels per group must stay exactly as it is now.

Reply:
please put comments about it